docs/usage.rst

```diff
@@ -0,0 +1,98 @@
+*****
+Usage
+*****
+
+Run Hyperparameter Search
```

Comment:
```suggestion
Run Batch of Jobs
```

Alternatively, "Run Array of Jobs" (more fitting to the cluster terminology, but probably less familiar for people)